### ISSUE TITLE
Add ability to leave a comment via Element Picker

### ISSUE THREAD

**Issue by sherlocknieh** [uBlockOrigin/uBlock-issues#1575]
Sometimes I need to keep tracking what I have just blocked by commenting below/above the rule line. 
It's annoying that i have to go to the setting page to add a comment.
So it woud be better if i can simply add a comment at the hover box (as the picture shows), please!

![屏幕截图 2021-05-02 105302](https://user-images.githubusercontent.com/43476643/116800785-bcdb3b80-ab36-11eb-83a1-1268c263620d.jpg)

**Comment by gwarser** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/456
